ci/scripts/docs_build.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -ex
+
+arrow_dir=${1}
+build_dir=${2}
+
+export LD_LIBRARY_PATH=${ARROW_HOME}/lib:${LD_LIBRARY_PATH}
+export PKG_CONFIG_PATH=${ARROW_HOME}/lib/pkgconfig:${PKG_CONFIG_PATH}
+export GI_TYPELIB_PATH=${ARROW_HOME}/lib/girepository-1.0
+export CFLAGS="-DARROW_NO_DEPRECATED_API"
+export CXXFLAGS="-DARROW_NO_DEPRECATED_API"
+
+# Sphinx
+sphinx-build -b html ${arrow_dir}/docs/source ${build_dir}/sphinx
+
+# C++
+pushd ${arrow_dir}/cpp/apidoc
+OUTPUT_DIRECTORY=${build_dir}/cpp doxygen
+popd
+
+# C GLib
+pushd ${arrow_dir}/c_glib
+mkdir -p ${build_dir}/c_glib/build
+pushd ${build_dir}/c_glib/build
+meson --prefix=$ARROW_HOME --libdir=lib -Dgtk_doc=true . ${arrow_dir}/c_glib
+ninja
+# TODO(kszucs): HTML output is not generated
```

Comment:
@kou Ported it to Debian 10, but the html output is not generated using meson. What am I missing?

Reply:
The default `all` "build" doesn't include "build"s to build documents. So we need to run "build"s to build documents explicitly or run `install` "build" that includes "build"s to build documents.

I think that the latter is better because we may add more "build"s to build documents.

How about the following?

```bash
ninja install
rsync -a --delete ${ARROW_HOME}/share/gtk-doc/html/* ../
```